ambry-server/src/main/java/com/github/ambry/server/AmbryServerRequests.java

```diff
@@ -536,6 +548,56 @@ private AdminResponse handleBlobIndexRequest(DataInputStream requestStream, Admi
         errorMessage.getBytes());
   }
 
+  /**
+   * Handles {@link ForceDeleteAdminRequest}.
+   * @param requestStream the serialized bytes of the request.
+   * @param adminRequest the {@link AdminRequest} received.
+   * @return the {@link AdminResponse} to the request.
+   */
+  private AdminResponse handleForceDeleteRequest(DataInputStream requestStream, AdminRequest adminRequest) {
```

Comment:
let's add a configuration to turn on and off this admin request, we want to turn this by default in prod.

Reply:
Added.